lib/SystemStatistics.php

```diff
@@ -179,4 +184,28 @@ protected function getAppsInfo() {
 		return $info;
 	}
 
+	/**
+	 * Checks if a function is available. Borrowed from
+	 * https://github.com/nextcloud/server/blob/2e36069e24406455ad3f3998aa25e2a949d1402a/lib/private/legacy/helper.php#L475
+	 *
+	 * @param string $function_name
+	 * @return bool
+	 */
+	public function is_function_enabled($function_name) {
+		if (!function_exists($function_name)) {
+			return false;
+		}
+		$disabled = explode(',', $this->phpIni->get('disable_functions') ?: '');
+		$disabled = array_map('trim', $disabled);
```

Comment:
I would like to suggest a small change here. IniGetWrapper provide a [listContains](https://github.com/bantuXorg/php-ini-get-wrapper/blob/master/src/IniGetWrapper.php#L120) function and does exactly what 198-202 does.

`if ($this->phpIni->listContains('disable_functions', $function_name))` should do the trick.